app/assets/stylesheets/active_admin/components/_status_tags.scss

```diff
@@ -6,10 +6,6 @@
   padding: 3px 5px 2px 5px;
   font-size: 0.8em;
 
-  &.ok, &.published, &.complete, &.completed, &.green { background: #8daa92; }
-  &.warn, &.warning, &.orange { background: #e29b20; }
-  &.error, &.errored, &.red { background: #d45f53; }
-
```

Comment:
Yes that's fine. We can deprecate the CSS. I'll have to be careful when rebasing on the css overhaul that it doesn't get lost. I will still keep the CSS in the changelog though and note that its deprecated so if users want it they should copy it themselves.